rust/agama-server/src/web/http.rs

```diff
@@ -34,6 +34,19 @@ use pam::Client;
 use serde::{Deserialize, Serialize};
 use utoipa::ToSchema;
 
+#[derive(Serialize, ToSchema)]
+pub struct LogsResponse {
+    /// Logs archive file name
+    logs: String,
+}
+
+#[utoipa::path(get, path = "/logs", responses(
+    (status = 200, description = "Compressed Agama logs", body = LogsResponse)
+))]
+pub async fn logs() -> Json<LogsResponse> {
+    Json(LogsResponse { logs: "/path/to/file".to_string() })
```

Comment:
There is already [a way to get the logs](https://github.com/agama-project/agama/blob/master/rust/agama-server/src/manager/web.rs#L234).

Reply:
well, i was not aware of this piece.  I knew, we have some logs downloading in the web UI but remembered only an old "hotfix" implementation

But in my POV the code you reference to needs to be redesigned because it exposes a HTTP API which runs CLI "agama logs store" at the backend. Purpose of this PR was to separate frontend (in this case CLI) from actual generating of logs at the backend (to make possible to connect the CLI  to a remote backend). So, the code you pointed doesn't need to use "agama logs store" anymore, but can use new HTTP API call "http://.../api/logs/store" directly.

Reply:
I'm not sure whether you want the change I described above in this PR or separate one.

Reply:
My point is that we have `/api/manager/logs` and `/api/logs`. My expectation was to fix the first one to do not use the command instead of creating another one.

Reply:
yeah. As said, I overlooked that /api/manager/logs, just rememebered an old implementation. So, I'll add fix for this one and try to merge  manager's http_client any here added logs' http_client ... should not be a big deal, just moving the code around.